src/elements/fields/TextField.jsx

```diff
@@ -1,15 +1,47 @@
-import { IMaskMixin } from 'react-imask';
+import { IMaskInput } from 'react-imask';
 import React, { memo } from 'react';
-import ReactForm from 'react-bootstrap/Form';
+
 import InlineTooltip from '../components/Tooltip';
 import { bootstrapStyles } from '../styles';
 import { emailPatternStr } from '../../utils/formHelperFunctions';
 
-function getTextFieldProps(servar, styles, value) {
-  let methods, onlyPhone;
+const MAX_TEXT_FIELD_LENGTH = 512;
+
+function escapeDefinitionChars(str) {
```

Comment:
reason for this?

Reply:
some imask characters are special, e.g. * means any char allowed, 0 means any digit allowed, etc. this makes them treated literally